### PR TITLE
LPAL-1053 Remove use of fpassthru in PHP code

### PR DESCRIPTION
**Pull request by townxelliot**
## Purpose

[LPAL-1053]()

## Approach

See ticket.

## Learning

n/a

## Checklist

* [X] I have performed a self-review of my own code
* [X] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [X] I have added tests to prove my work
* [X] I have added mandatory tags to terraformed resources, where possible
* [X] If I have a new OPG component dependency, I have updated the `metadata.json` with the repo location.
* [X] If I added a package.json or composer.json, I also made sure this is included in the script in `.github/workflows/dependabot-update.yml`
* [ ] The product team have tested these changes

### CODE REVIEW

File: cypress/e2e/Admin.feature [hunk 1]
```diff
@@ -142,3 +132,6 @@ Feature: Admin
         | test-no-email-no-number |
         | test-no-num             |
         | test                    |
+
+    # LPAL-1053: check feedback export to CSV works correctly
+    And I can export feedback and download it as a CSV file
```
Comment (nickdavis2001) [inline cypress/e2e/Admin.feature:137]: Given that the way of generating the csv has been completely changed (which is good :) ). should there be a test that ensures the contents of the csv are OK?  Assuming we didn't have one before...  which we should have of course.  Might be better as unit test?

Comment (townxelliot) [reply cypress/e2e/Admin.feature:137]: There are very few unit tests on any of this code. I could write a test to cover this relatively easily, I think.

Comment (townxelliot) [reply cypress/e2e/Admin.feature:137]: Not easy to unit test as it manipulates headers and writes directly to the output stream. I tried a work-around which is meant to work for this, but couldn't get it to work. I'll extend the cypress test instead.

Comment (townxelliot) [reply cypress/e2e/Admin.feature:136]: @nickdavis2001 I have added some additional checks of the structure of the data, if you could please re-review. Doing this in a unit test proved tricky, as testing handlers (with their csrf tokens etc.) was unwieldy and unintuitive.

**Review by nickdavis2001 — APPROVED** [review]

**Pull request merged**